server/util/bazel/defs.bzl

```diff
@@ -6,23 +6,57 @@ def _extract_bazel_installation_impl(ctx):
         outputs = [out_dir],
         inputs = [ctx.executable.bazel],
         command = """
-            USER=nobody "$BAZEL" --output_user_root="$OUT_DIR/.output_user_root" --install_base="$OUT_DIR"/.install --output_base="$OUT_DIR"/.output &>action.log
-            code="$?"
-            trap 'rm action.log' EXIT
-            if (( code )); then
-                cat action.log >&2
-                exit "$code"
-            fi
-
             set -e
-            rm -rf "$OUT_DIR"/.output_user_root
-            rm -rf "$OUT_DIR"/.output
-            mv "$OUT_DIR"/.install/* "$OUT_DIR"/
-            rmdir "$OUT_DIR"/.install
+
+            BAZEL="$PWD/$BAZEL"
+            OUT_DIR="$PWD/$OUT_DIR"
+            WORKDIR="$PWD/_extract_bazel_installation_tmp"
+
+            INSTALL_BASE="$OUT_DIR/install_base"
+            REPOSITORY_CACHE="$OUT_DIR/repository_cache"
```

Comment:
From repo cache flag description:

> An empty string as argument requests the cache to be disabled, otherwise the default of '<output_user_root>/cache/repos/v1' is used

So I think it should already be in out_dir by default?